app/assets/stylesheets/home.css.scss

```diff
@@ -5,48 +5,51 @@
 $splash-image-height: 600px;
 
 .splash-block {
-    @include full-width-block($off-white);
-    background-color: transparent;
+  @include full-width-block($off-white);
+  background-color: transparent;
 }
 
 .splash-images {
-    z-index: -20;
-    margin-top: $topbar-height + 1;
-    position: absolute;
-    width: 100%;
-    top: 0;
-    .splash-image {
-        background-size: cover;
-        height: $splash-image-height;
-    }
+  z-index: -20;
+  margin-top: $topbar-height + 1;
+  position: absolute;
+  width: 100%;
+  top: 0;
+	
+  .splash-image {
+    background-size: cover;
+    height: $splash-image-height;
+  }
 }
 
 .splash-overlay {
-    z-index: -10;
-    margin-top: $topbar-height + 1;
-    position: absolute;
-    width: 100%;
-    height: $splash-image-height;
-    top: 0;
-    background-color: rgba(0, 0, 0, 0.2);
-
+  z-index: -10;
```

Comment:
Properties should be ordered background-color, height, margin-top, position, top, width, z-index